packages/create-plugin/src/commands/generate.command.ts

```diff
@@ -175,9 +184,15 @@ async function generateFiles({ actions }: { actions: any[] }) {
         path: action.path,
       });
     } catch (error) {
+      let message;
+      if (error instanceof Error) {
+        message = error.message;
+      } else {
+        message = String(error);
+      }
```

Comment:
Did you receive some errors due to the previous implementation?

Reply:
Yeah, I think enabling `strict` via the tsconfig changes caused this to error. Error is typed as `unknown` so `error.message || error.toString()` were causing complaints. Dunno if that helps explain it better?

Reply:
Yea, absolutely, thanks for the explanation 👍